### DISCUSSION TITLE
responsive below 480px

### DISCUSSION THREAD

**Discussion by KostandinosVas** [tailwindlabs/tailwindcss#9503] [Help]
Ok here is the thing. I am trying to make my project responsible for mobile but below 480px min-width it seems like not working. I changed the min-width in the tailwindconfig and nothing changed at all. Is there something i am missing?

**Comment by wongjn** [comment]
Could you elaborate further on what seems like is "not working"? When you say you "changed the min-width in the tailwindconfig" – what did you actually change? What did you want to happen when you changed this configuration? Perhaps reproduce what you currently have in [Tailwind Play](https://play.tailwindcss.com/) so we can see more closely the problems you are currently experiencing

**Reply by KostandinosVas** [reply]
![ab](https://user-images.githubusercontent.com/82898197/194600089-481b8472-22ee-42c7-8422-d78e7dc23a6e.png)
![tailconf](https://user-images.githubusercontent.com/82898197/194600129-46c5c4e0-574b-4298-92e3-27d589c4040a.png)
So i want the width of the homepage to be 100%. Until 500px it is full as i want it but under 500 it leaves empty space on the right.

**Reply by KostandinosVas** [reply]
![Screenshot_1](https://user-images.githubusercontent.com/82898197/194600594-b1b90272-f224-4cc9-95fb-509ab81868d2.png)
and this is over 575 px

**Reply by wongjn** [reply]
You could add a custom `screen` configuration between your `sm` and `md` like:
```diff
 screens: {
   'sm': '300px',
   // => @media (min-width: 576px) { ... }
+
+  'foo': '500px',
   
   'md: '960px',
   // => @media (min-width: 960px) { ... }
```
And then you can use this custom screen variant like so:

```html
<div class="w-full foo:w-1/2">
  <!-- Full width until 500px, then 50% width -->
</div>
```

**Reply by KostandinosVas** [reply]
Seems like a good solution but still the whitespace is there. Thank you very much i will try to rebuild the project to see maybe i've done something very wrong in the whole project
